streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1609,12 +1609,11 @@ public void shouldReturnStateManagerChangelogOffsets() {
     }
 
     @Test
-    public void shouldCheckpointWithCreatedStateOnClose() {
+    public void shouldNotCheckpointOnCloseCreated() {
```

Comment:
Why did this test change?

Reply:
It was the same as the other one, that it was wrong to begin with.